tmt.spec

```diff
@@ -448,7 +448,7 @@ install -pm 644 %{name}/steps/provision/mrack/mrack* %{buildroot}/etc/%{name}/
 - Always show the real beaker job id
 - Create a production copr build for each release
 - AVC denials check for tests (#2331)
-- Add nice & colorfull help to "make" targets
+- Add nice & colorful help to "make" targets
```

Comment:
Is it possible to apply these? If not we can either:
- Add an `exclude-file` to match the exact lines
- Convert to `%autochangelog` + packit generated changelog from github release page

Reply:
I believe the autogenerated changelog was discussed last week during the tmt hacking session, and it was decided against this particular automation.

Reply:
> it was decided against this particular automation.

I believe the consensus was to try to let packit generate changelog?

Reply:
Aha, I might be wrong then. My impression was the automation was undesirable.

Reply:
There might be a bit of a setup necessary to make sure the generate release notes are compatible [^1]. Particularly `.github/release.yaml` generates bullet points with `*` instead of `-`. Maybe in either case a custom action is needed to sanitize that.

[^1]: https://packit.dev/docs/configuration#copy_upstream_release_description

Reply:
Will be resolved by #2880.